WalletWasabi/Announcements/ReleaseHighlights.md

```diff
@@ -1,49 +1,38 @@
 ## Release Highlights
-#### 🤫 Send to Silent Payment addresses<br/>
-#### 🎁 Donation Button<br/>
-#### 🧪 ~~TestNet 3~~ -> TestNet 4<br/>
-#### 📝 Release Notes in the client<br/>
+#### 💪 3rd Party Providers for Fee & Exchange Rate<br/>
+#### 👨‍🔧 Quality of Life Features<br/>
+#### 📦 Backend and Coordinator packaged for Linux<br/>
 
 ## Release Summary
-Wasabi Wallet v2.4.0 brings send to Silent Payment addresses!
+Wasabi Wallet v2.5.0 continues to enhance Wasabi's resilience and improve the user interface to reduce frustration.
 
-### 🤫 Send to Silent Payment addresses
+### 💪 3rd Party Providers for Fee & Exchange Rate
 
-Silent Payments are privately reusable static addresses. While sending Bitcoin to them is as easy as sending to regular addresses, it is not possible for an observer to link several payments to the same Silent Payments address.
+Fee rate estimations and exchange rate providers are now configurable.
 
-They are useful to send an address to multiple persons at the same time (tips, donations...) or to make multiple payments between the same entities (salary, rent...).
+Previously, this information was provided by Wasabi's backend. This change aligns with our long-term goal of completely removing dependence on the backend.
 
-Wasabi is now one of the first wallet to support send to Silent Payment addresses, and we hope that more wallets will join! We are working on allowing receiving, please bear with us!
+After this release, Wasabi will be significantly closer to achieving that goal, with most basic wallet features functioning without an active backend connection. You can read more about the remaining steps [here](https://github.com/orgs/WalletWasabi/discussions/13661).
```

Comment:
nit
for the paranoid user,
it is not clear that clicking on the link will open the browser